### PR TITLE
Add chat via gitter

### PR DESCRIPTION
**Pull request by steffansluis**
Add chat-message and chat-room model, chat-user-picture component
and some other things

### CODE REVIEW

**Comment by steffansluis** [comment]
@bartist @WouterKvG Please review!

File: addon/components/chat-window.js [hunk 1]
```diff
@@ -7,11 +7,11 @@ export default Component.extend({
...
   classNames: 'chat-window',
   session: service(),
+  account: service(),
+  ajax: service(),
 
   openChat: false,
-  windowSize: 'chat-closed',
-  contentChat: true,
-  messages: [],
+  windowSize: 'chat-open',
```
Comment (bartist) [inline addon/components/chat-window.js]: I think this should be closed, so you don't start with an open chat-window

**Comment by bartist** [comment]
twajxit

**Comment by WouterKvG** [comment]
out of date

File: addon/components/chat-window.js (hunk 1 above)
Comment (WouterKvG) [inline addon/components/chat-window.js]: Agree

**Pull request merged**